src/mappers/world/world_mapper.cpp

```diff
@@ -8,26 +8,22 @@
 
 namespace mappers
 {
-/// <summary>
-/// Create a world_mapper from file info.
-/// </summary>
-/// <returns></returns>
-WorldMapper WorldMapper::LoadFromFiles(commonItems::ModFilesystem hoi4_mod_filesystem, const vic3::World& source_world)
+
+WorldMapperBuilder WorldMapperBuilder::DefaultMapper(commonItems::ModFilesystem hoi4_mod_filesystem,
```

Comment:
Is there a way we can do this without having to supply a ModFilesystem? or have a default ModFilesystem that we can use?

Reply:
It *is* needed for the province mapper to double-check all HoI4 provinces were mapped. If trying to create a default one for tests, I just usually do something like

```DefaultMapper(commonItems::ModFilesystem("", {}), test_source_world)```
or
```DefaultMapper(commonItems::ModFilesystem("<folder with relevant test files>", {}), test_source_world)```

Reply:
unknown file: error: C++ exception with description "Could not find /map/definition.csv" thrown in the test body.

Reply:
I get this error when I try an empty file path. 
what is the correct filepath to stop this error?

Reply:
You'll need one with test files that include /map/definition.csv. ```test_files/hoi4_world``` should do it (the data/test_files folder gets copied to the test_files folder next to the test binary).